languages/javascript/concepts/nullability/about.md

```diff
@@ -1 +1,70 @@
-TODO: add information on nullability concept
+## Nullability
+
+The key thing to remember about Null is that it is a primitive value, used to represent an intentional absence of an
```

Comment:
As the concept documents are now standalone and no longer part of the exercise, maybe rephrase this to:

```suggestion
Null is a primitive value, used to represent an intentional absence of an
```